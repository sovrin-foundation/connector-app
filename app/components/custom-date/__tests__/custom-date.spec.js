// @flow
import React from 'react'
import 'react-native'
import renderer from 'react-test-renderer'
import CustomDate from '../custom-date'

describe('<CustomDate />', () => {
  it('should ConnectionHistoryDetails render properly', () => {
    const component = renderer.create(
      <CustomDate>{'2018-01-10T04:32:43+05:30'}</CustomDate>
    )
    expect(component).toMatchSnapshot()
  })
})
