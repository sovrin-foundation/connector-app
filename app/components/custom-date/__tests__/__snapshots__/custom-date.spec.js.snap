// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomDate /> should ConnectionHistoryDetails render properly 1`] = `
<Text
  accessible={true}
  allowFontScaling={false}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "fontSize": 17,
      },
      Object {
        "fontFamily": "Lato",
      },
      Object {
        "backgroundColor": "#333333",
        "color": "#ffffff",
      },
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
    ]
  }
>
  2018-01-10T04:32:43+05:30
</Text>
`;
